extensions/kubernetes-config/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/AbstractKubernetesConfigSourceUtil.java

```diff
@@ -72,6 +72,7 @@ abstract ConfigSource createPropertiesConfigSource(String kubernetesConfigSource
             } else {
                 // TODO all keys named `*.{properties,yml,yaml}` are categorized as file sources,
                 //  but here, we only look for `application.{properties,yml,yaml}`
+                // Also see qualifier in categorize function below
```

Comment:
The `TODO` is no longer valid, we should remove it, or replace with an up-to-date text.